src/pe_reports/stakeholder/views.py

```diff
@@ -527,6 +522,7 @@ def getalluserinfo():
 @stakeholder_blueprint.route("/stakeholder", methods=["GET", "POST"])
 def stakeholder():
     """Process form information, instantiate form and render page template."""
+    LOGGER.info("made it to stakeholder")
```

Comment:
Same as above regarding `info` vs. `debug`.

Reply:
@dav3r 
The logger category has been changed as you have mentioned. See commit [d11f89c](https://github.com/cisagov/pe-reports/pull/169/commits/d11f89c78939b26ed8344d0ad355b62febfc21d8)